planet/api/client.py

```diff
@@ -19,7 +19,7 @@
 from .exceptions import (InvalidIdentity, APIException)
 from . import models
 from .utils import check_status
-
+from retrying import retry
```

Comment:
ohh sorry, I meant to pull this against my own fork. I will open a proper PR here when I get this figured out.